build.sbt

```diff
@@ -144,7 +144,7 @@ sparkEmrConfigs := List(
     "spark.default.parallelism" -> "960", //Testing 960 partitions for 106 Grids at ~2GB/grid slawler
     "spark.rdd.compress" -> "true",
     "spark.driver.extraJavaOptions" -> "-Djava.library.path=/usr/local/lib",
-    "spark.executor.extraJavaOptions" -> "-XX:+UseParallelGC -Dgeotrellis.s3.threads.rdd.write=64 -Djava.library.path=/usr/local/lib",
+    "spark.executor.extraJavaOptions" -> "-XX:+UseParallelGC -Djava.library.path=/usr/local/lib",
```

Comment:
Removing the overwrite for number of threads to use, this is potentially aggressive, again does not seem like the error was happening on the write side because the checker board pattern was matching input images, not output grid.